bats.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: bats
   version: 1.11.0
-  epoch: 1
+  epoch: 0
```

Comment:
I think this should be 2, otherwise existing one will be pulled only unless you've removed the r0 one. 💭

Reply:
You are right, I think I reset it midway. Let me change that.